AABB_tree/include/CGAL/internal/AABB_tree/AABB_ray_intersection.h

```diff
@@ -180,7 +180,14 @@ class AABB_ray_intersection {
 
   struct as_ray_param_visitor {
     typedef FT result_type;
-    as_ray_param_visitor(const Ray* ray) : ray(ray) {}
+    as_ray_param_visitor(const Ray* ray)
+     : ray(ray), max_i(0)
+    {
+      typename AABB_traits::Geom_traits::Vector_3 v = ray->to_vector();
+      for (int i=1; i<3; ++i)
+        if( CGAL::abs(v[i]) > CGAL::abs(v[max_i]) )
+          max_i = i;
+    }
```

Comment:
That does not look like a doc fix. Was it supposed to be in this PR?

Reply:
An extra commit slipped in. Fixed.